salalib/tidylines.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include "salalib/spacepix.h"
+
+// helpers... a class to tidy up ugly maps people may give me...
+
+class TidyLines : public SpacePixel
+{
+public:
+   TidyLines() {;}
```

Comment:
I am not sure I like the shape of this class. If I understand correctly, you have to construct it with the standard constructor, and then feed the data via the `tidy` or `quicktidy` methods which will turn this into a valid SpacePixel.
Wouldn't it be better if we had two classes that took the data in the constructor and constructed a tidy lines SpacePixel on construction?